src/Components/CategoryCard/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState} from 'react';
+import './style.component.css'
+import Dropdown from './dropdown'
+
+
+function CategoryCard({title, imageUrl}) {
+    const [active, setActive] = useState(false);
+
+
+    function toggle(){
+        setActive(!active)
+
+    }
+
+    return (
+        <div className={`${active ? "category-card" : "category-card-white"}`} >
+            <button className={`button-container ${active}`} onClick={toggle}>
+            <div className="image-container">
+                <div className="circle">
```

Comment:
Evitar colocar propriedades nos nomes das classes

```suggestion
                <div className="images">
```